src/openzaak/conf/includes/base.py

```diff
@@ -658,3 +658,9 @@
 
 # Name of the cache used to store responses for requests made when importing catalogi
 IMPORT_REQUESTS_CACHE_NAME = config("IMPORT_REQUESTS_CACHE_NAME", "import_requests")
+
+# settings for uploading large files
+MIN_UPLOAD_SIZE = int(os.getenv("MIN_UPLOAD_SIZE", 4 * 2 ** 30))
+CHUNK_SIZE = 4 * 2 ** 30  # 4 GB
+READ_CHUNK = 6 * 2 ** 20  # 6 MB
+DEFAULT_EXTENSION = "bin"
```

Comment:
let's properly namespace these settings

```suggestion
# settings for uploading large files
DOCUMENTEN_UPLOAD_CHUNK_SIZE = 4 * 2 ** 30  # 4 GB
DOCUMENTEN_UPLOAD_READ_CHUNK = 6 * 2 ** 20  # 6 MB
DOCUMENTEN_UPLOAD_DEFAULT_EXTENSION = "bin"
```

`MIN_UPLOAD_SIZE` is already defined in the section 'Custom settings', and that's also where these settings belong. Please stick to the existing structure/grouping of settings.